src/effects/native/reverbeffect.cpp

```diff
@@ -99,11 +99,21 @@ void ReverbEffect::processChannel(const ChannelHandle& handle,
     Q_UNUSED(enableState);
     Q_UNUSED(groupFeatures);
 
+    if (!pState || !m_pSendParameter || !m_pBandWidthParameter || !m_pDecayParameter || !m_pDampingParameter) {
+        qWarning() << "Could not retrieve all effect parameters";
+        return;
+    }
+
     const auto bandwidth = m_pBandWidthParameter->value();
     const auto decay = m_pDecayParameter->value();
     const auto damping = m_pDampingParameter->value();
     const auto send = m_pSendParameter->value();
 
-    pState->reverb.setSampleRate(sampleRate);
+    if (!pState->didInit) {
+        // init() function for reverb needs sampleRate
+        // this doesn't allow changing the sampleRate but it doesn't strongly affect the sound
```

Comment:
I tested and it does unexpectedly shift the pitch of the reverberations when changing the sample rate until Mixxx is restarted. That's not a terrible bug, but it should be avoided if possible.